tests/test_prometheus.py

```diff
@@ -0,0 +1,37 @@
+"""Tests for the Prometheus containers."""
+
+import pytest
+import requests
+from pytest_container.container import ContainerData
+
+from bci_tester.data import PROMETHEUS_CONTAINERS
+from bci_tester.data import ALERTMANAGER_CONTAINERS
+from bci_tester.data import BLACKBOX_CONTAINERS
+
+
+ready_containers = PROMETHEUS_CONTAINERS + ALERTMANAGER_CONTAINERS
+healthy_containers = (
```

Comment:
Can you convert these to constants please (i.e. all caps)? Also, I am not really sure what the names should tell me. Imho all containers that we ship should be healthy by default and why should the blackbox_exporter not be ready?

```suggestion
ready_containers = PROMETHEUS_CONTAINERS + ALERTMANAGER_CONTAINERS
healthy_containers = (
```